pytest_fixtures/component/settings.py

```diff
@@ -1,19 +1,20 @@
 # Settings Fixtures
 import pytest
-from nailgun import entities
 
 
-@pytest.fixture(scope="function")
-def setting_update(request):
+@pytest.fixture()
+def setting_update(request, target_sat):
     """
     This fixture is used to create an object of the provided settings parameter that we use in
     each test case to update their attributes and once the test case gets completed it helps to
     restore their default value
     """
-    setting_object = entities.Setting().search(query={'search': f'name={request.param}'})[0]
+    setting, value_to_change = request.param
```

Comment:
There are many tests using this fixture and they dont return tuple, so those tests will fail here with unpacking as we will be simply unpacking single string into tuple.

Handle the unpacking and further process based on parameters please.

Reply:
ah, right, I just updated the test to not pass key value as tuple instead I'm now passing key=value and for tests which don't pass value we can keep value as none, and no value gets updated by this fixture as earlier.
Please re-review let me know if you still see any issues with this.